### DISCUSSION TITLE
MPB - Australia!

### DISCUSSION THREAD

**Discussion by TheSchoolofBitcoinCurriculum** [MyFirstBitcoin/.github#27] [LNN Introductions]
## The School of Bitcoin (Australia)

We run Bitcoin initiatives here in Australia through our charity, **[Digital Playhouse Foundation LTD](https://www.digitalplayhouse.org.au/)**. Meet the team: Kieran Nolan, Electra Frost, and Gordon Christian.

### **The School of Bitcoin - V4V Open Lessons**

We hold our **monthly Value 4 Value lessons** on HiveTalk. These are interactive sessions where everyone can pitch in, learn, and share Bitcoin knowledge. You can drop your lightning address or Nostr pubkey to get direct tips from other learners. [Lessons](https://v4v.theschoolofbitcoin.com)

### **Decentralised Autonomous Education System (DAES)**

[Decentralised Autonomous Education System (DAES)](https://geyser.fund/project/theschoolofbitcoin)
The DAES learning community provides a platform for aspiring learners to submit their Bitcoin project ideas for potential funding and mentorship. By leveraging the decentralized nature of DAES, we create an inclusive environment where innovative ideas can thrive. We also manage and moderate the [Stacker News /~Education territory](https://stacker.news/~Education), where learners can submit their projects for consideration. Additionally, we offer a [Signal](https://signal.group/#CjQKIDweK1LO2ox37j-KbVBSpc12XQjQpmVFEZihZ5RcNnEzEhC-P2ADBmgEVCjsU4qp8ndc) chat for collaboration and mentorship, enhancing our support for promising proposals. Anyone interested in supporting these projects can contribute to our [learner fund](https://geyser.fund/project/theschoolofbitcoin), helping to bring innovative ideas to fruition.

**Comment by kdnolan** [comment]
# Hello! I'm Kieran:

I'm passionate about innovation, disruption, and lifelong learning. You can find me at [kdnolan.com](https://kdnolan.com), or reach out to me via Nostr: `npub1cyla8qgt9gv8y6ydv8s2prt89h8afc0sr2kaz64ryjmlpdrzxm4qwlh53q`.

## Current Roles and Projects

- **Learning Programs Director** at the Digital Playhouse Foundation
  - Dedicated to digital literacy projects such as [theschoolofbitcoin.com](https://theschoolofbitcoin.com)
- **Community Program Organizer** on the Mornington Peninsula
- **Host** of the #getoffzero podcast, documenting Bitcoin learning journeys

## Teaching Experience

- **Teaching Bitcoin in Schools** since 2013
  - Contributing to a transformative education approach
- **Educational Technologist** at Wooranna Park Primary School
  - Pioneered Victoria's first CoderDojo
  - Established Australia's first Immersive Education Club
- **STEM Wizard** at Boneo Primary School

## Recognitions and Media Features

- **Speaker** at prestigious conferences worldwide
- **Educator Hotlist** in 2019 and 2020
- Featured in media outlets such as:
  - The George Lucas Foundation
  - Bitcoin.com
  - The Age
  - Australian Teacher Magazine

I'm dedicated to pushing the boundaries of learning and technology.

**Comment by electrafrost** [comment]
Thanks @kdnolan for the intros.
To clarify - Kieran, Gordon and I run theschoolofbitcoin.com project in Australia under the Digital Playhouse Foundation umbrella. 

Hi everyone, I'm Electra Frost, I'm an accountant working with small to medium size businesses using bitcoin and crypto since 2013. My passion is showing business owners and other accountants how to adopt and build on Bitcoin, and I'm working on education projects for the accounting industry.

I'm also a director and CFO of Digital Playhouse Foundation which we set up for running Bitcoin-focused projects through in 2021, for educating regional communities and showing self-employment pathways. We care about protecting our communities from scams and schemes, so educating from a Bitcoin-first POV is important for anyone entering the crypto space. Due to crypto scams and misinformation about Bitcoin, a lot of people around here who could be benefitting from bitcoin are missing out. Once they learn Bitcoin, and how to use and secure it, they can make better informed decisions about everything else and/or decide to just stick with Bitcoin. That's the goal.

In 2022 I visited El Salvador for Adopting Bitcoin, and used the MPB English translation for our local workshops. We would like to run a Light Node for MPB in Agnes Water to bring MBP to more people in regional Queensland. Connections in Townsville and Cairns (further north) will follow.

Gordon will introduce himself next. We met when he led the project in our town as Australia's first "Bitcoin beach" in 2018 as a tourism initiative.
https://www.abc.net.au/news/2018-06-03/australias-first-digital-currency-friendly-town/9828306
